### ISSUE TITLE
Commonly used image buttons FAIL 1.4.5: Images of Text?

### ISSUE THREAD

**Issue by jake-abma** [w3c/wcag#1748]
A common pattern for 'get it on Google Play, or Apple Store" are image buttons used with logo's in it, or also brand fonts, but also with some extra background and extra text, makes the button NOT logo's.

So in this case, will they all be failures for WCAG?
(I have to say "yes" for the complete button image, but you'll never know...)

<img width="295" alt="Screen Shot 2021-04-20 at 11 31 44 AM" src="https://user-images.githubusercontent.com/6384002/115373321-0d17dc80-a1cc-11eb-8441-985668a4604c.png">

From Deque demo site: https://dequeuniversity.com/demo/dream

**Comment by patrickhlauke** [comment]
With the usual caveat that this is all about the vagueness of 1.4.5 (see https://github.com/w3c/wcag/issues/1647 for the more recent discussion on this), I'd say those do not fail as it's not really easy to  achieve the same exact laid-out look and feel with the custom fonts etc using other technologies (though of course everything IS possible if you spend enough time on it...you can redo the Mona Lisa using nothing but a collection of CSS-styled `<div>` elements if you really want to...but there's some cut-off point of where it can be reasonably expected from an author to do so).

I mean sure, they *could* be done as `<svg>` elements too, I guess (see also https://github.com/w3c/wcag/issues/773) but pragmatically, that's an unrealistic ask for authors as well in most cases.

**Comment by patrickhlauke** [comment]
x-ref also https://github.com/w3c/wcag/issues/1614

**Comment by jake-abma** [comment]
I see your Point @patrickhlauke but at the same time you stretch the definition for logo, or..., extending the exception to:

"Logo's OR a form logo's are used, and possible brand fonts, together in combination with graphical objects and background effects, to indicate a link or button, or something else, all is fine the moment we all start using it and becomes 'our way we always do it'"

**Comment by scottaohara** [comment]
I agree with where you're coming from here, but per the normative language these sorts of images would fall under "essential"... because they are specific branding images that authors have no control over by design.  You _have_ to use these images to link to the app store / play store, with no modifications.

**Comment by patrickhlauke** [comment]
no, not claiming that the whole button is a logo, so not stretching the definition. i'm saying that "the technologies being used *can't* (easily) achieve the visual presentation" (unless we expect authors to mess about with trying to load the correct corporate webfont, do some positioning shenanigans to get the button to align and look just like the marketing folks wanted it, etc)

**Comment by bruce-usab** [comment]
Is anyone arguing that these two button examples should fail against 1.4.5?  (Regardless, I am happy that @jake-abma raised the question.)

**Comment by patrickhlauke** [comment]
@bruce-usab  Pragmatically, no. But that depends on where one lands on the "is it practical" vs "is it technically achievable" scale when evaluating the "If the technologies being used can achieve the visual presentation" part...

**Comment by bruce-usab** [comment]
Thanks @patrickhlauke.  I agree it would be good to have some consensus language providing exposition around <q>technically achievable</q>.  I have had content managers argue that that adding `alt` is not technically feasible (and might actually have asserted <q>undue burden</q> because their CMS did not provide for that when adding an image to a page.  (This was years ago.)

**Comment by patrickhlauke** [comment]
I believe this issue retreads the same ground as https://github.com/w3c/wcag/issues/1647 and https://github.com/w3c/wcag3/issues/190
